projects/1.16/assets/brewevolution/brewevolution/lang/zh_cn.json

```
@@ -0,0 +1,77 @@
+{
+  "item.brewevolution.rum_glass_item": "杯装朗姆酒",
+  "item.brewevolution.vodka_glass_item": "杯装伏特加",
+  "itemGroup.tabbrewevolution_creative_tab": "酿造进化",
+  "block.brewevolution.full_mug": "满大杯",
+  "subtitles.burp": "玩家：打嗝",
+  "block.brewevolution.calvados_bottle": "瓶装卡尔瓦多斯",
+  "block.artofbeer.empty_mug": "空大杯",
+  "item.brewevolution.liqueur_bottle_item": "瓶装甜浆果利口酒",
+  "block.artofbeer.full_mug": "满大杯",
+  "block.brewevolution.calvados_glass": "杯装卡尔瓦多斯",
+  "item.brewevolution.horn_item": "角杯",
+  "block.brewevolution.rhum_bottle": "瓶装朗姆酒",
+  "block.artofbeer.hops_crop_stage_6": "啤酒花作物：阶段6",
+  "block.artofbeer.hops_crop_stage_7": "啤酒花作物：阶段7",
+  "block.brewevolution.sweet_berry_liqueur_glass": "杯装甜浆果利口酒",
+  "block.artofbeer.hops_crop_stage_4": "啤酒花作物：阶段4",
+  "block.brewevolution.beer_sign": "啤酒标志",
+  "item.brewevolution.calvados_bottle_item": "瓶装卡尔瓦多斯",
+  "block.artofbeer.hops_crop_stage_5": "啤酒花作物：阶段5",
+  "block.artofbeer.hops_crop_stage_2": "啤酒花作物：阶段2",
+  "block.artofbeer.hops_crop_stage_3": "啤酒花作物：阶段3",
+  "item.brewevolution.full_mug_item": "大杯装啤酒",
+  "block.artofbeer.hops_crop_stage_0": "啤酒花作物：阶段0",
+  "block.artofbeer.hops_crop_stage_1": "啤酒花作物：阶段1",
+  "item.brewevolution.vodka_bottle_item": "瓶装伏特加",
+  "block.brewevolution.hops_crop_stage_0": "啤酒花作物：阶段0",
+  "item.artofbeer.hops_seeds": "啤酒花种子",
+  "block.brewevolution.rum_glass": "杯装朗姆酒",
+  "item.brewevolution.dried_bread": "面包干",
+  "item.brewevolution.mollasses": "糖蜜",
+  "item.brewevolution.mead_horn": "角杯装蜜糖酒",

```

```suggestion
  "item.brewevolution.mead_horn": "角杯装蜂蜜酒",
```
mead跟上面那个molasses没有关系，molasses是用来做朗姆酒的